mkdocs.yml

```diff
@@ -90,6 +90,12 @@ nav:
     - Line marks: demo/line-marks.md
     - Status line: demo/statusline.md
     - Input modes: input-modes.md
+  - VT extensions:
```

Comment:
`font-settings.md` is missing from the list, just pointing this our in case you forgot to add it in here

Reply:
done